### PR TITLE
Webpack stuff tj

### LINKED ISSUE #74: use Webpack to bundle page
As a dev I want my page to be complied using webpack

### PR DESCRIPTION
**Pull request by taylorjohnson141**
## Description
This adds the capability to use web-pack for the site

Clearly and concisely describe the feature.
The site now relies on webpack to work

## Affected areas of application
index.html , webpack config, domupdates dist, 
List all parts of the website that will be altered from this PR.
Pantry no longer works and the images won't load 
## How Has This Been Tested?
npm install
npm run-script build
npm start

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Test A

- Test B

## Relevant Tickets
#74 
What tickets/issues were resolved by this feature

closes #74 

## Screenshots
Screen Shot 2020-09-13 at 8 52 23 PM

Add screenshots here if applicable

### CODE REVIEW

**Comment by nicolegooden** [comment]
Hey @taylorjohnson141 what happened to the images?  Were they unable to show after webpack was implemented?  Thank you for investigating this a bit more.  I'm looking forward to discussing your process tomorrow!

File: src/recipe.js [hunk 1]
```diff
@@ -49,4 +49,4 @@ class Recipe {
   }
 }
 
-// export default Recipe;
+export default Recipe;
```
Comment (nicolegooden) [inline src/recipe.js:52]: I'm so happy these are back up and running!

File: src/scripts.js [hunk 2]
```diff
@@ -1,213 +1,172 @@
...
-// import Pantry from './pantry';
-// import Recipe from './recipe';
-// import User from './user';
-// import Cookbook from './cookbook';
-// import domUpdates from './domUpdates';
-
+import './css/base.scss';
+import './css/styles.scss';
+
+import recipeData from './data/recipes';
```
Comment (nicolegooden) [inline src/scripts.js:4]: We can get rid of these imports from the data files since we are using fetch (lines 4-6)

**Review by nicolegooden — APPROVED** [review]

**Review by CalebCyphers — APPROVED** [review]
Thanks for spending the time to figure this out. I was wondering what the significance of the 'dist' folder was!

**Pull request merged**